src/firmware/boards/robot_stm32h7/freertos.c

```diff
@@ -320,29 +377,94 @@ void initIoNetworking(void)
+    io_ublox_odinw262_communicator_init(&huart4, ublox_reset_pin, 2);
 
-    // initialize network logger
+    // Initialize network logger
     io_network_logger_init(RobotLogProtoQHandle);
 }
 
 void initIoDrivetrain(void)
 {
-    // TODO (#2063) the motor driver pins are not wired correctly on mainboard rev2.0
-    // We will not be trying to drive any motors until we have mainboard rev2.1 which
-    // addresses the board issues.
+    // MOTOR WHEEL_BACK_LEFT
+    GpioPin_t *motor_a_reset_pin = io_gpio_pin_create(
+        WHEEL_BACK_LEFT_RESET_GPIO_Port, WHEEL_BACK_LEFT_RESET_Pin, ACTIVE_HIGH);
+    GpioPin_t *motor_a_mode_pin = io_gpio_pin_create(
+        WHEEL_BACK_LEFT_MODE_GPIO_Port, WHEEL_BACK_LEFT_MODE_Pin, ACTIVE_HIGH);
+    GpioPin_t *motor_a_dir_pin = io_gpio_pin_create(WHEEL_BACK_LEFT_DIR_GPIO_Port,
+                                                    WHEEL_BACK_LEFT_DIR_Pin, ACTIVE_HIGH);
+
+    // MOTOR WHEEL_FRONT
+    GpioPin_t *motor_b_reset_pin = io_gpio_pin_create(
+        WHEEL_FRONT_LEFT_RESET_GPIO_Port, WHEEL_FRONT_LEFT_RESET_Pin, ACTIVE_HIGH);
+    GpioPin_t *motor_b_mode_pin = io_gpio_pin_create(
+        WHEEL_FRONT_LEFT_MODE_GPIO_Port, WHEEL_FRONT_LEFT_MODE_Pin, ACTIVE_HIGH);
+    GpioPin_t *motor_b_dir_pin = io_gpio_pin_create(
+        WHEEL_FRONT_LEFT_DIR_GPIO_Port, WHEEL_FRONT_LEFT_DIR_Pin, ACTIVE_HIGH);
+
+    // MOTOR C
+    GpioPin_t *motor_c_reset_pin =
+        io_gpio_pin_create(DRIBBLE_RESET_GPIO_Port, DRIBBLE_RESET_Pin, ACTIVE_HIGH);
+    GpioPin_t *motor_c_mode_pin =
+        io_gpio_pin_create(DRIBBLE_MODE_GPIO_Port, DRIBBLE_MODE_Pin, ACTIVE_HIGH);
+    GpioPin_t *motor_c_dir_pin =
+        io_gpio_pin_create(DRIBBLE_DIR_GPIO_Port, DRIBBLE_DIR_Pin, ACTIVE_HIGH);
+
+    // MOTOR D
+    GpioPin_t *motor_d_reset_pin = io_gpio_pin_create(
+        WHEEL_BACK_RIGHT_RESET_GPIO_Port, WHEEL_BACK_RIGHT_RESET_Pin, ACTIVE_HIGH);
+    GpioPin_t *motor_d_mode_pin = io_gpio_pin_create(
+        WHEEL_BACK_RIGHT_MODE_GPIO_Port, WHEEL_BACK_RIGHT_MODE_Pin, ACTIVE_HIGH);
+    GpioPin_t *motor_d_dir_pin = io_gpio_pin_create(
+        WHEEL_BACK_RIGHT_DIR_GPIO_Port, WHEEL_BACK_RIGHT_DIR_Pin, ACTIVE_HIGH);
+
+    // MOTOR E
+    GpioPin_t *motor_e_reset_pin = io_gpio_pin_create(
+        WHEEL_FRONT_RIGHT_RESET_GPIO_Port, WHEEL_FRONT_RIGHT_RESET_Pin, ACTIVE_HIGH);
+    GpioPin_t *motor_e_mode_pin = io_gpio_pin_create(
+        WHEEL_FRONT_RIGHT_MODE_GPIO_Port, WHEEL_FRONT_RIGHT_MODE_Pin, ACTIVE_HIGH);
+    GpioPin_t *motor_e_dir_pin = io_gpio_pin_create(
+        WHEEL_FRONT_RIGHT_DIR_GPIO_Port, WHEEL_FRONT_RIGHT_DIR_Pin, ACTIVE_HIGH);
+
+    PwmPin_t *motor_a_pwm_pin = io_pwm_pin_create(&htim15, TIM_CHANNEL_2);
+    PwmPin_t *motor_b_pwm_pin = io_pwm_pin_create(&htim3, TIM_CHANNEL_2);
+    PwmPin_t *motor_c_pwm_pin = io_pwm_pin_create(&htim1, TIM_CHANNEL_3);
+    PwmPin_t *motor_d_pwm_pin = io_pwm_pin_create(&htim8, TIM_CHANNEL_1);
+    PwmPin_t *motor_e_pwm_pin = io_pwm_pin_create(&htim1, TIM_CHANNEL_2);
```

Comment:
can we rename these to not be a,b,c,d,e?

Reply:
fixed